src/components/common/Extensions/UserNavigation/partials/WalletConnectedTopMenu/WalletConnectedTopMenu.tsx

```diff
@@ -32,13 +37,27 @@ const WalletConnectedTopMenu: FC<UserAvatarDetailsProps> = ({
           isVerified={isVerified}
         />
       </div>
-      <Link
-        to={`${USER_HOME_ROUTE}/${USER_EDIT_PROFILE_ROUTE}`}
-        className="navigation-link hover:bg-gray-50 rounded -ml-4 w-[calc(100%+2rem)]"
-      >
-        <Icon name="user-circle-gear" appearance={{ size: iconSize }} />
-        <p className="ml-2">{formatMessage({ id: 'userMenu.menageTitle' })}</p>
-      </Link>
+      {user ? (
+        <Link
+          to={`${USER_HOME_ROUTE}/${USER_EDIT_PROFILE_ROUTE}`}
+          className="navigation-link hover:bg-gray-50 rounded -ml-4 w-[calc(100%+2rem)]"
+        >
+          <Icon name="user-circle-gear" appearance={{ size: iconSize }} />
+          <p className="ml-2">
+            {formatMessage({ id: 'userMenu.manageTitle' })}
+          </p>
+        </Link>
+      ) : (
+        <Link
+          to={`${CREATE_PROFILE_ROUTE}`}
```

Comment:
We don't need the template string here, we can write it like this: `to={CREATE_PROFILE_ROUTE}`

Reply:
Totally right, removed this! :)